gossip/c_event_callbacks.go

```diff
@@ -196,12 +196,15 @@ func (s *Service) processEvent(e *inter.EventPayload) error {
 	}
 
 	newEpoch := s.store.GetEpoch()
+	newEpochNotEqualOld := newEpoch != oldEpoch
```

Comment:
You can name "sealed" or "epochSealed". "Epoch is sealed" means that it had ended and a next epoch has started